config/routes.rb

```diff
@@ -4,6 +4,14 @@
     namespace :v1 do
       resources :merchants
       resources :items
+
+      namespace :merchants do
+        get '/:id/items', to: 'items#index'
+      end
+
+      namespace :items do
+        get '/:id/merchant', to: 'merchants#show'
+      end
```

Comment:
Utilized more nested namespacing to hit desired route. Used hand rolled URI as resources would not work correctly.